www/templates/common/trip-list.html

```diff
@@ -1,6 +1,6 @@
-<ion-view nav-title="Trips list">
-  <ion-nav-title class="dark-color">
-    <!--
+<ion-view title="Trips list">
+<!--   <ion-nav-title class="dark-color">
```

Comment:
just so I understand what is happening here - we want the title to be "Trips list". And it is right there in as `title="Trips list"`. But the `<ion-nav-title>` overrides the`title` param and then is all blank anyway. So you re-introduce it here.

Looks good!